packages/atomic/src/components/search/atomic-layout/search-layout.spec.ts

```diff
@@ -3,6 +3,11 @@ import {AtomicLayoutSection} from '../../common/atomic-layout-section/atomic-lay
 import {AtomicSearchLayout} from './atomic-search-layout';
 import {buildSearchLayout} from './search-layout';
 
+/* eslint-disable prettier/prettier */
+// TODO: get rid of the mock once no longer running stencil tests
```

Comment:
No need to add jira ticket as this will eventually have to be replaced during lit migration